packages/bloc/lib/src/cubit.dart

```diff
@@ -176,7 +174,6 @@ abstract class Cubit<State> extends Stream<State> {
   Future<void> close() {
     // ignore: invalid_use_of_protected_member
     _observer.onClose(this);
-    _controller ??= StreamController<State>.broadcast();
```

Comment:
As a side question: why is it necessary for a not fully initialized `Cubit` (without the controller being set) to emit 'done' event? If it is uninitialized, it means nobody ever subscribed to it, so nobody cares whether 'done' is emitted or not, right? I know there are some tests for it but as per my understanding it looks like they are contrived, test something that would never occur outside of tests and in the real world nobody would wait on the 'done' event since nobody ever subscribed (otherwise, if anybody subscribes, the controller is initialized and this line in `close` will have no effect whatsoever and 'done' will be emitted).

What am I missing? What is the use case for emitting 'done' even without any subscribed being important?